.github/actions/coverage/action.yml

```diff
@@ -6,6 +6,9 @@ inputs:
   env-shop:
     description: "Environment code"
     required: true
+  env-dashboard:
```

Comment:
**Issue**: The new 'env-dashboard' input is added but lacks a default value which might lead to issues if not provided. <br> **Fix**: Provide a default value for 'env-dashboard' to ensure the workflow runs smoothly even if the value is not provided externally. <br> **Code Suggestion**: 
 ```
 +    default: "default-dashboard-env"
 ```